cloudsdk/src/main/java/io/particle/android/sdk/cloud/exceptions/ParticleCloudException.java

```diff
@@ -154,6 +155,26 @@ public String getServerErrorMsg() {
         return serverErrorMessage;
     }
 
+    /**
+     * Server-provided multi-factor auth token.  May be null.
+     *
+     * @return server-provided multi-factor auth token or null
+     */
+    public String getMfaToken() {
```

Comment:
We're already using GSON everywhere else. Why the inconsistency?

Reply:
Adding a specialized auth method onto a generic API exception doesn't feel great to me.  Could this specialized logic be handled elsewhere (somewhere that isn't exposed as public API surface)?

Reply:
Note 'loadServerErrorMsg' method, we infact do not use gson here, so this is consistent with the rest of the class.

Reply:
I believe it has value as public API, we could however handle it elsewhere or have child exception, something like 'ParticleLoginException'.